hatchet_sdk/contracts/dispatcher_pb2.pyi

```diff
@@ -1,9 +1,16 @@
+# type: ignore
+
+from typing import ClassVar as _ClassVar
```

Comment:
did you add these yourself, these are autogenerated so it'll be overwritten on next gen

Reply:
cool, sounds good - these should be safe then 👍

Reply:
alright just excluded those files so the diff is a lot smaller!